cli/lsp/testing/execution.rs

```diff
@@ -131,14 +122,14 @@ fn as_test_messages<S: AsRef<str>>(
 }
 
 #[derive(Debug, Clone, Default, PartialEq)]
-struct TestFilter {
-  maybe_include: Option<HashMap<String, TestDefinition>>,
-  maybe_exclude: Option<HashMap<String, TestDefinition>>,
+struct LspTestFilter {
+  include: Option<HashMap<String, TestDefinition>>,
+  exclude: HashMap<String, TestDefinition>,
```

Comment:
Why is exclude now not optional?

Reply:
This is another redundant `Option`, blocklists are inherently non-optional because that's the same thing as being empty